### PR TITLE
fix: rkaiq: add Conflicts and Replaces to control

### PR DESCRIPTION
**Pull request by CodeChenL**
To fix upgrade failed

### CODE REVIEW

**Comment by CodeChenL** [comment]
@RadxaYuntian request a review

**Review by RadxaYuntian — APPROVED** [review]

**Comment by RadxaYuntian** [comment]
CI报错

File: fixup/fix_rkaiq [hunk 1]
```diff
@@ -27,7 +27,11 @@ fixup_callback() {
     soc="$(sed -E "s/.*(rk[0-9]+).*/\1/" <<< "$file_name")"
     package_name="$(grep "Package:" control | cut -d' ' -f2)"
     sed -i -E "s/^(Version: .*)$/\1-$soc/" control
-    echo "Provides: $package_name-$soc" >> control
+    cat >> control <<EOF
```
Comment (RadxaYuntian) [inline fixup/fix_rkaiq]:
```suggestion
    cat <<EOF >> control 
```
顺序稍微翻转一下，这样接收端就是在最末尾

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
改完直接重写一下提交历史

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**